pkg/executables/executables.go

```diff
@@ -167,3 +178,17 @@ func execute(ctx context.Context, cli string, in []byte, args ...string) (bytes.
 	}
 	return stdout, nil
 }
+
+func (e *linuxDockerExecutable) executeCleanup(ctx context.Context, containerName string) {
+	dockerCommands := []string{
+		"rm", "-f", "-v", containerName,
+	}
+	cmd := exec.CommandContext(ctx, "docker", dockerCommands...)
+	infoMessage := "cleaning up container " + containerName
+	logger.V(6).Info(infoMessage, "cmd", cmd)
+
+	err := cmd.Run()
+	if err != nil {
+		logger.V(0).Error(err, "error cleaning up docker container")
```

Comment:
If the verbosity is 0, do we need to specify this here? Also can we display what container failed to clean up here? Because of that, wondering if we should do verbosity of 1 instead.